__tests__/universal/routes.spec.jsx

```diff
@@ -44,17 +47,37 @@ describe('routes', () => {
 
   beforeEach(() => jest.clearAllMocks());
 
-  it('should set up the root module route first', () => {
-    const RootRoute = createRoutes(store)[0];
-    expect(ReactTestUtils.isElement(RootRoute)).toBe(true);
-    expect(RootRoute.props).toEqual({ moduleName: 'fakeRootModule', store });
+  describe('RootModule without childroutes', () => {
+    it('should add a path on the root route', () => {
+      mockHasChildRoutesValue = false;
+      const RootRoute = createRoutes(store)[0];
+      expect(RootRoute.props.path).toBe('/');
```

Comment:
this test case appears to be completely covered by your second test.
Same for you third and fourth test.